setup.py

```diff
@@ -13,6 +13,7 @@
 try:
     from Cython.Distutils.extension import Extension
     from Cython.Distutils import build_ext
+    from Cython.Build import cythonize
```

Comment:
cythonize should not be used here. Need to commit pre-built cython sources (the .c files) directly or the source files will get re-generated on each build which may not work depending on the system doing the building.